src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,20 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
     public int getUserScore(String[] records, String email) {
-        return 0;
+        int userScope = -1;
+        for (String user : records) {
+            if (user.matches(email + ".*") && !email.matches(".*[0-9]")) {
+                String[] userRecords = user.split(":");
+                userScope = Integer.parseInt(userRecords[1]);
```

Comment:
if we've find user, we don't need to run loop anymore we can return value here

Reply:
done